From 2a5a4126a32dfa41e0039eeb45d0d42ec1ce56c1 Mon Sep 17 00:00:00 2001
From: Marc Kelly <mkelly@thegoodpenguin.co.uk>
Date: Wed, 10 Sep 2025 15:57:01 +0100
Subject: [PATCH 6/7] init/main.c Revert accidental change

Signed-off-by: Marc Kelly <mkelly@thegoodpenguin.co.uk>
---
 init/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/main.c b/init/main.c
index 30538810fc29..0ee0ee7b7c2c 100644
--- a/init/main.c
+++ b/init/main.c
@@ -894,7 +894,7 @@ static void __init early_numa_node_init(void)
 }
 
 asmlinkage __visible __init __no_sanitize_address __noreturn __no_stack_protector
-void `(void)
+void start_kernel(void)
 {
 	char *command_line;
 	char *after_dashes;
-- 
2.43.0

